doc/api/util.md

```diff
@@ -179,6 +179,10 @@ formatted string:
  - `customInspect` - if `false`, then custom `inspect(depth, opts)` functions
    defined on the objects being inspected won't be called. Defaults to `true`.
 
+ - `maxArrayLength` - specifies the maximum number of Array and TypedArray
+   elements to include when formatting. Defaults to `100`. Set to `Infinity`
+   to show all Array elements. Set to 0 or negative to show no array elements.
```

Comment:
tiny nits: Maybe capitalize `Array elements` consistently in this line, and enclose the constant 0 in backticks?